c/jwk.c

```diff
@@ -280,6 +288,12 @@ static void getPublicKey(Json *jwk, x509_public_key_info *publicKeyOut, int *sta
   if (!keyObject) {
     zowelog(NULL, LOG_COMP_ID_JWK, ZOWE_LOG_WARNING, "JWK doesn't contain key\n");
     *statusOut = JWK_STATUS_UNRECOGNIZED_FMT_ERROR;
+
+    zowelog(NULL, LOG_COMP_ID_JWK, ZOWE_LOG_WARNING, "JWK response:\n");
+    //Often enough, the destination has some error message that can be printed
+    jsonPrinter *jp = makeJsonPrinter(STDOUT_FILENO);
```

Comment:
@1000TurquoisePogs , are we sure there is no sensitive information at this point?